assets/js/common/ActivityLogDetailsModal/ActivityLogDetailModal.jsx

```diff
@@ -20,6 +20,72 @@ import {
   USER_CREATION,
   USER_DELETION,
   USER_MODIFICATION,
+  // Host events
+  HEARTBEAT_FAILED,
+  HEARTBEAT_SUCCEEDED,
+  HOST_CHECKS_HEALTH_CHANGED,
+  HOST_CHECKS_SELECTED,
+  HOST_DEREGISTERED,
+  HOST_DEREGISTRATION_REQUESTED,
+  HOST_DETAILS_UPDATED,
```

Comment:
Possible opportunity to evaluate openapi front-end code generation here?

Reply:
Could be an option indeed. Let's see how it evolves.